src/main/java/org/cbioportal/cdd/service/ClinicalDataDictionaryService.java

```diff
@@ -29,5 +29,5 @@
     List<ClinicalAttributeMetadata> getClinicalAttributeMetadata(String cancerStudy);
     List<ClinicalAttributeMetadata> getMetadataByColumnHeaders(String cancerStudy, List<String> columnHeaders) throws ClinicalAttributeNotFoundException;
     ClinicalAttributeMetadata getMetadataByColumnHeader(String cancerStudy, String columnHeader) throws ClinicalAttributeNotFoundException;
-    void forceResetCache();
+    List<String> forceResetCache();
```

Comment:
is this a list of strings because we are planning to add additional messages to what's returned?

Reply:
ah, I see

Reply:
was just curious. code looked good to me